docs/errors.md

```diff
@@ -207,17 +159,9 @@ The classes to be tested will be selected based on two characteristics:
 - Attempt to execute `tryNext()` on a `ChangeStream` object that is closed.
   - Assert that `MongoStreamClosedError` is thrown.
 
-#### `MongoTopologyClosedError`
-
-- Attempt to execute `createCollection()` against a database that has been closed.
-  - Assert that `MongoTopologyClosedError` is thrown.
-
-#### `MongoCursorExhaustedError`
-
-- Attempt to continue reading a cursor after it has reached the end of the batch.
-  - Assert that `MongoCursorExhaustedError` is thrown.
+### `MongoRuntimeError`
```

Comment:
If nothing's being tested under MongoRuntimeError, we can remove it from the md.